addons/editor-dark-mode/addon.json

```diff
@@ -1002,6 +1002,34 @@
         "darkScrollbars": true
       }
     },
+    {
+      "name": "Scratch 2.0 Style",
+      "id": "scratch2",
+      "description": "A theme based on Scratch 2.0's colors.",
+      "values": {
+        "page": "#ffffffff",
+        "primary": "#179fd7ff",
+        "highlightText": "#4d97ffff",
```

Comment:
Scratch 2.0 uses orange (`#fba939`) for hovered menu bar items and block categories. I think that should be the highlight text color.

Reply:
Hmm... it looks weird on all other places... 
![image](https://user-images.githubusercontent.com/72760579/132947626-67e4f4a0-23d8-47c9-9d20-9a033ca0c4e9.png)

Reply:
What about `#1e9ed6` (used for hovered and selected icons)?